modules/db-writer/src/main/resources/analytics.sql

```diff
@@ -21,13 +21,16 @@ create table if not exists executed_deposit (
     stake_pkh Text,
     collateral_ada BIGINT not null,
     order_input_id Text not null,
-    user_output_id Text not null,
-    pool_input_Id Text not null,
-    pool_output_Id Text not null,
-    timestamp BIGINT not null
+    user_output_id Text,
```

Comment:
As improvement you can regroup tables fields like:
coin_x, amount_x, coin_y, amount_y 
This allows us to fetch not flat but nested classes from DB like:
`case class Asset(coin: String, amount: Long)`
`case class Deposit(assetX: Asset, assetY: Asset)`
`select coin_x, amount_x, coin_y, amount_y from deposit.query[Deposit]`